framework/stats/hwcpipe_stats_provider.cpp

```diff
@@ -90,31 +94,37 @@ HWCPipeStatsProvider::HWCPipeStatsProvider(std::set<StatIndex> &requested_stats)
 	{
 		switch (iter->second.type)
 		{
-			case StatType::Cpu:
-			{
+			case StatType::Cpu: {
```

Comment:
Why has the `{` moved off of its own line?

Reply:
I used clang format 15 to format the code as this is the version we now use in the CI (latest version). It's likely that we need to tweak our clang format style file. Is this a blocker for the PR or something we can fix later?

Reply:
You probably just need to add
`     AfterCaseLabel : true`
in the `BraceWrapping` section in the .clang_format file.
Besides that: no, I wouldn't consider that as a blocker.

Reply:
Thanks! that worked